### DISCUSSION TITLE
ImportError: cannot import name 'track_iter_progress' from 'mmcv' (unknown location)

### DISCUSSION THREAD

**Discussion by Jiayiiiiiii502** [open-mmlab/mmcv#2946] [Q&A]
I have already installed the mmcv-full, but there is still an import error. Can anyone help please?

pip list | grep mm
comm                      0.1.4
mmcv-full                 1.4.0
mmdet                     2.14.0
mmdet3d                   0.17.2
mmsegmentation            0.14.1
timm                      0.9.5

Traceback (most recent call last):
  File "tools/create_data.py", line 6, in <module>
    from data_converter.create_gt_database import create_groundtruth_database
  File "/root/autodl-tmp/MapTR/tools/data_converter/create_gt_database.py", line 5, in <module>
    from mmcv import track_iter_progress
ImportError: cannot import name 'track_iter_progress' from 'mmcv' (unknown location)

**Comment by Random-junhao** [comment]
I have encountered the same problem as you. Have you resolved this issue? Can you please let me know how it was resolved? Thank you very much!

**Comment by LinyeLyuNeo** [comment]
change 
from mmcv import track_iter_progress
to 
from mmcv.utils.progressbar import track_iter_progress
works for me
